.editorconfig

```diff
@@ -0,0 +1,198 @@
+###############################
+# Core EditorConfig Options   #
+###############################
+
+root = true
+
+# All files
+[*]
+charset = utf-8
+indent_style = tab
+tab_width = 8
+trim_trailing_whitespace = true
+max_line_length = 180
+
+# MSBuild
+[*.{csproj,proj,projitems,shproj,fsproj,targets,props}]
+indent_style = space
+indent_size = 2
+
+# XML config files
+[*.{xml,axml,config,nuspec,resx}]
```

Comment:
`xaml` too? We have a Xamarin.Forms test app.

Reply:
Added!